src/components/OldMeetups/OldMeetups.js

```diff
@@ -80,67 +79,67 @@ export function OldMeetups ({ meetups }) {
-                      style={{ backgroundImage }}
-                    />
-                  </div>
+              <div className={styles.hideOnMobile}>
+                <div className={styles.meetup__date}>
+                  <FullDate date={meetup.date} fontSize={.5} />
+                  <div
+                    className={styles.meetup__image}
+                    role="img"
+                    aria-label="image relative au sujet du meetup"
+                    style={{ backgroundImage }}
+                  />
                 </div>
-                <div className={styles.meetup__content}>
-                  <div className={styles.meetup__header}>
-                    <h4
-                      className={styles.meetup__title}
-                      title={meetup.title}
-                    >
-                      {meetup.title}
-                    </h4>
-                    <div className={styles.meetup__sponsors}>
-                      <p>
-                        {meetup.sponsor && (
-                          <Fragment>
-                            <span>
-                              Sponsorisé par <b>{meetup.sponsor.name}</b>
-                            </span>
-                            <br />
-                          </Fragment>
-                        )}
-                        {meetup.venue && (
-                          <span>Hébergé par <b>{meetup.venue.name}</b></span>
-                        )}
-                      </p>
-                    </div>
+              </div>
+              <div className={styles.meetup__content}>
+                <header className={styles.meetup__header}>
+                  <h4
+                    className={styles.meetup__title}
+                    title={meetup.title}
+                  >
+                    {meetup.title}
+                  </h4>
+                  <div className={styles.meetup__sponsors}>
+                    <p>
+                      {meetup.sponsor && (
+                        <Fragment>
+                          <span>
+                            Sponsorisé par <b>{meetup.sponsor.name}</b>
+                          </span>
+                          <br />
+                        </Fragment>
+                      )}
+                      {meetup.venue && (
+                        <span>Hébergé par <b>{meetup.venue.name}</b></span>
+                      )}
+                    </p>
                   </div>
-                  <Talks talks={meetup.talks}/>
-                  <div className={styles.showOnMobile}>
-                    <div className={styles.meetup__date}>
-                      <p>{meetup.date}</p>
-                    </div>
+                </header>
+                <Talks talks={meetup.talks} />
+                <div className={styles.showOnMobile}>
+                  <div className={styles.meetup__date}>
+                    <p>
+                      <time dateTime={formatRaw(meetup.date)}>
```

Comment:
this is the important part, the rest of the file ( except the first `section` ) is indented